spec/jobs/voice_otp_sender_job_spec.rb

```diff
@@ -14,15 +14,21 @@
 
       calls = FakeVoiceCall.calls
 
-      code = '1234'.scan(/\d/).join(', ')
-      message = t('jobs.voice_otp_sender_job.message', code: code)
-      url_message = URI.escape(message)
-
       expect(calls.size).to eq(1)
       call = calls.first
       expect(call.to).to eq('555-5555')
-      expect(call.url).to include(url_message)
       expect(call.from).to match(/(\+19999999999|\+12222222222)/)
+
+      code = '1234'.scan(/\d/).join(', ')
+      query = Rack::Utils.parse_nested_query(URI(call.url).query)
+      expect(query['Message']).to eq(t('jobs.voice_otp_sender_job.message_repeat', code: code))
+
+      nested_query = query
+      4.times do
```

Comment:
is this the max? if so, maybe assign to a var to make test clearer? If not, why pick 4?